server/lib/system/system.getGladysContainerId.js

```diff
@@ -42,7 +42,15 @@ async function getGladysContainerId() {
       // then, we remove .scope
       [containerId] = firstPart.split('.scope');
     } else {
-      throw new PlatformNotCompatible('DOCKER_CGROUP_CONTAINER_ID_NOT_AVAILABLE');
+      const mountinfo = await fs.promises.readFile('/proc/self/mountinfo', 'utf-8');
```

Comment:
What happens if this file doesn't exist? should be properly catched :)

Reply:
Done!! 😄 

I took the liberty to refactor the method a little bit and split it into 3 functions: `getContainerIdFromCidfile`, `getContainerIdFromCgroup` and `getContainerIdFromMountinfo`. So if it's not found in one file, goes to the next one until it's found or throws an Error.

Reply:
Looks way better now :) Thanks for the refacto, much cleaner now!